recipe/meta.yaml

```diff
@@ -1,69 +1,123 @@
+{% set version = "1.13.1" %}
+
+{% set pyver = "cp{}{}".format(PY_VER[0], PY_VER[2]) %}
+{% set platform = "macosx_10_11_x86_64" %}  # [osx]
+{% set platform = "win_amd64" %}  # [win]
+{% set fn = "{}-{}-{}-{}m-{}.whl".format("tensorflow", version, pyver, pyver, platform) %}  # [not linux]
+{% set fn = "tensorflow-{{ version }}.tar.gz" %}  # [linux]
+
 package:
   name: tensorflow
-  version: "1.13.1"
+  version: {{ version }}
+
+source:
+  fn: {{ fn }}
+  url: https://github.com/tensorflow/tensorflow/archive/v{{ version }}.tar.gz  # [linux]
+  sha256: 7cd19978e6bc7edc2c847bce19f95515a742b34ea5e28e4389dade35348f58ed  # [linux]
+
+  url: https://pypi.io/packages/{{ pyver }}/t/tensorflow/{{ fn }}  # [not linux]
+  sha256: 46fc216db780b1a7b7a28cf6536d6ec0a171ef2fa546a24934ffe077c734c891  # [osx and py27]
+  sha256: a3851118d2cc5e9e697503753c2d247dca0e98d5a85b5ea091103f27d9e57a38  # [osx and py36]
+  sha256: 2fc989a5d3be9f3dc96c20deaa3d5cc66d2bf76fe069c202c2914a7f4b5a9a35  # [osx and py37]
+
+  sha256: d7341617aedd73c2c847755e87697e9c19eb625c73da26d6cd669220c5565119  # [win and py36]
+  sha256: a4f154fa98cde5974117e05af444e0bed84cf431b8129c78a2de03a1ff7201c9  # [win and py37]
+
+  patches:
+    - patches/0001-add-rt-to-linkopts.patch  # [linux]
+    - patches/0002-stable-image-rotate-test.patch  # [linux]
+    - patches/0003-relax-tolerence-in-special_math_test.patch  # [linux]
+    - patches/0004-skip-read-only-debugger_cli_common-test.patch  # [linux]
+    - patches/0005-fix-lookup_ops-test.patch  # [linux]
+    - patches/0006-relax-precision-in-matrix_log_op_test.patch  # [linux]
 
 build:
-  number: 0
-  #  Google supplies whl files on PyPI for:
-  # - Linux: Python 2.7, 3.3, 3.4, 3.5 and 3.6
-  # - OS X: Python 2.7, 3.3, 3.4, 3.5 and 3.6
-  # - Win-64: Python 3.5 and 3.6
-  skip: true  # [win]  # Skip all windows for now due to numpy/scipy version pinning issues
-  skip: True  # [win32]
-  skip: True  # [win64 and py<35]
-  skip: True  # [osx and py34]
-  skip: True  # [py26]
-  skip: True  # [py33]
+  number: 1
+  skip: True  # [not x86_64]
+  skip: True  # [win and vc<14]
+  script: "{{ PYTHON }} -m pip install --no-deps -vv {{ fn }}"  # [not linux]
```

Comment:
This does not work and fails with:

```
Found a build.sh script and a build/script section inside meta.yaml.
Either remove the build.sh script or remove the build/script section in meta.yaml.
```

I have a vague memory of ``conda-build`` allowing this in the past, and not sure what are the alternatives. Multiple outputs have the `install_script` and single output have `script` and `build script`, but we don't have an unified way to use those together as far as I know.


@jjhelmus any suggestions?

Reply:
One idea is to remove build.sh completely and split it out into `build_linux.sh` and `build_osx.sh`.  Then the script section can have a different entry for each platform.